spec/support/dynamoid_helper.rb

```diff
@@ -0,0 +1,16 @@
+module DynamoidReset
+  def self.all
+    Dynamoid.adapter.list_tables.each do |table|
+      # Only delete tables in our namespace
```

Comment:
Is it possible for this constraint to be enforced outside of the application code?

Reply:
Which constrain sorry? This shouldn't be too much of an issue as it just a database cleaner for when we run the tests.